splash/qtrender_lua.py

```diff
@@ -328,7 +329,7 @@ def __call__(self, *args):
             args=escape_js(*args)
         )
         wrapper_script = get_process_errors_js(expr)
-        res = self.tab.evaljs(wrapper_script)
+        res = self.tab.evaljs(wrapper_script, result_protection=self.js_result_protection)
```

Comment:
@kmike what do you mean? Should new rules be added to [sanitize](https://github.com/scrapinghub/splash/blob/master/splash/jsutils.py#L12) function? Or it should be written like
```js
var rect = elem. getBoundingClientRect();
return { top: rect.x, right: rect.right, bottom: rect.bottom, left: rect.left };
```